static/admin.js

```diff
@@ -72,13 +76,15 @@ let message_handlers = {
 
 admin_feed.onmessage = e => {
     let data = JSON.parse(e.data)
-    console.log(data)
     if ('topic' in data && 'data' in data) {
+        console.log(data)
```

Comment:
Do we still want this?

Reply:
Can be useful, but the Firefox web socket support has got better. I would keep it for now, it's only on the admin page anyway